### ISSUE TITLE
Dart 3.0 produces Error: The method 'NullThrownError' isn't defined for the class 'ResumableMediaUploader' when this package is used

### ISSUE THREAD

**Issue by a-siva** [google/googleapis.dart#478]
With the new Dart 3.0 sdk we get following errors when this package is used

```
Running command "/b/s/w/ir/cache/builder/src/out/host_release/dart-sdk/bin/dart --disable-dart-dev /b/s/w/ir/cache/builder/src/flutter/testing/benchmark/test/parse_and_send_test.dart"
../../../third_party/pkg/googleapis/discoveryapis_commons/lib/src/resumable_media_uploader.dart:77:44: Error: The method 'NullThrownError' isn't defined for the class 'ResumableMediaUploader'.
 - 'ResumableMediaUploader' is from 'package:_discoveryapis_commons/src/resumable_media_uploader.dart' 
```

Looks like this was fixed in https://github.com/google/googleapis.dart/pull/472

A new version of this package needs to be published.

**Comment by kevmoo** [comment]
published as 1.0.5!

**Comment by nosmirck** [comment]
I'm having the same issue when using packahe: `googleapis: ^11.2.0`
